FrostyCmd/Profiles/Fifa20.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Name": "FIFA20",
+  "DisplayName": "FIFA 20",
+  "InternalName": "fifa20",
+  "Deobfuscator": "Signature",
+  "TypeInfoSignature": "",
+  "DataVersion": 20190927,
+  "FrostbiteVersion": "3.0",

```

Comment:
i made a post in #developer in the discord which has the correct fb versions for all games